tools/Measure-Types.ps1

```diff
@@ -94,7 +94,7 @@ public static class Hook
 
     $resolveTargets = @(
         $TargetVersion
-        'netcoreapp2.0'
+        'netcoreapp3.1'

```

Comment:
I don't know if this script is actually used by anything any longer so it likely doesn't matter but it isn't clear if this change would actually work or not. At any rate I leave it to you if you want to undo or just keep it.